xml/System/Range.xml

```diff
@@ -270,16 +270,18 @@
       <Docs>
         <param name="length">The length of the collection that the range will be used with. <paramref name="length" /> has to be a positive value.</param>
```

Comment:
From https://github.com/dotnet/dotnet-api-docs/pull/2729, the wording was modified:

https://github.com/dotnet/dotnet-api-docs/pull/2729/files#diff-329e2957d7afa212df1c2d8a57ae415aR266

```text
      <Docs>
        <param name="length">A positive integer that represents the length of the collection that the range will be used with.</param>
        <summary>Returns a <see cref="T:System.Range.OffsetAndLength" /> instance with the starting offset and length of a range.</summary>
        <returns>The starting offset and length of the range.</returns>
        <remarks>
          <format type="text/markdown">
            <![CDATA[
## Remarks
For performance reasons, this method does't validate `length` to ensure that it is not negative. It does ensure that `length` is within the current `Range` instance. 
            ]]>
          </format>
        </remarks>
        <exception cref="T:System.ArgumentOutOfRangeException"><paramref name="length" /> is outside the bounds of the current range. </exception>
      </Docs>
```

Reply:
Thanks @ahsonkhan. @rpetrusha please check which version you prefer. The old or the one in this PR which is a mix of the current + old.

Reply:
I prefer the one in the PR, which is much clearer, @mairaw.